Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
     
-    func loginControllerDidTapOnRegister(controller: LoginController)
+    func loginControllerDidTapOnSignup(controller: LoginController)
```

Comment:
Couldn't we change this naming? I'd prefer `DidTapOnSignup(controller: LoginController)`. We are copying `UIKit` way of naming the delegates while they said in last WWDC that this naming would be improved.